gateway/modules/crud/sql/helpers.go

```diff
@@ -238,13 +238,17 @@ func (s *SQL) processJoins(ctx context.Context, query *goqu.SelectDataset, join
 			return nil, helpers.Logger.LogError(helpers.GetRequestID(ctx), fmt.Sprintf("Invalid join type (%s) provided", j.Type), nil, nil)
 		}
 
-		isValidJoin, columnName := utils.IsValidJoin(j.On, j.Table)
-		if isValidJoin {
-			sel[j.Table+"."+columnName] = 1
+		// Don't put any fields in select clause if it's an aggregation request
+		if !isAggregate {
```

Comment:
Does cache invalidation work when we are doing aggregation with joins? Preparing the join mapping can cause a problem I guess